code/datums/uplink_item.dm

```diff
@@ -580,7 +580,7 @@ var/list/uplink_items = list()
 	desc = "When inserted into a personal digital assistant, this cartridge gives you four opportunities to detonate PDAs of crewmembers who have their message feature enabled. \
 	The concussive effect from the explosion will knock the recipient out for a short period, and deafen them for longer. It has a chance to detonate your PDA."
 	item = /obj/item/weapon/cartridge/syndicate
-	cost = 6
+	cost = 2
```

Comment:
I have no idea what the chance for your PDA is to explode, but unless it's at a reasonable level then i'd prefer the cost being at 3, maybe 4.